docs/products/db/quickstart.mdx

```diff
@@ -60,15 +59,11 @@ class AppDatabase extends _$AppDatabase {
 
   @override
   int get schemaVersion => 1;
-}
 
-LazyDatabase _openConnection() {
-  return LazyDatabase(() async {
-    // Locally, Drift creates a file named 'my-first-db.db'.
+  static QueryExecutor _openConnection() {
     // On deployment, Globe automatically connects this to the live database.
-    final file = File('my-first-db');
-    return NativeDatabase(file);
-  });
+    return NativeDatabase.opened(sqlite3.open('your-db-name.db'));
+  }
```

Comment:
Is this change necessary?

Reply:
Yes, but it works. 

Used the previous approach because it's used more commonly in examples with drift (and the Flutter engineers are probably more familiar with this). However this also goes.